.travis.yml

```diff
@@ -1,12 +1,27 @@
 language: java
 sudo: true
+
 jdk:
+  - openjdk8
   - oraclejdk8
+  - openjdk11
+
+matrix:
+  allow_failures: 
+    - jdk: openjdk11
+  fast_finish: true
+
 before_install:  
  - "echo $JAVA_OPTS"
  - "export JAVA_OPTS=-Xmx512m"
 
-script:
+# Default installation command is 
+# mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+# this is what we test as our build phase so skip it here.
+install:
+ - true
```

Comment:
What does this directive do?

Reply:
Answering my own question: https://docs.travis-ci.com/user/job-lifecycle/#skipping-the-installation-phase